### ISSUE TITLE
ImageNet pretrained model

### ISSUE THREAD

**Issue by FangGet** [noahzn/Lite-Mono#4]
Hi, thanks for your great work and code.  I trained the Lite-Mono with no pretrained weights and get a similar performance as in the paper. But is it possible for you to upload the ImageNet pretrained model?

**Comment by noahzn** [comment]
Hi, I will upload the pre-trained weights soon and let you know.

**Comment by FangGet** [comment]
Thanks.

**Comment by FangGet** [comment]
Thanks @noahzn .
With AdamW optimizer, weight_decay=1e-2, drop_path=0.2 learning_rate=1e-4, pretrained Imagenet, I got the results:
| name| abs_rel |   sq_rel |     rmse | rmse_log |       a1 |       a2 |       a3 |
|  ----  | ----  | ----  | ----  | ----  | ----  | ----  | ----  |
|paper | 0.107  |   0.765  |    4.561  |    0.183  |    0.886  |    0.963  |    0.983 | 
| own_trained| 0.108  |   0.798  |  4.658  |   0.187  |   0.885  |   0.961  |   0.982|

Gap still exists, waiting for your training detail(maybe different learning rate for depth and pose network?).

**Comment by noahzn** [comment]
Hi! Glad to see that you trained the network by yourself! Some comments:

1. I used the initial learning rate `1e-4` for both the DepthNet and the PoseNet. Then, the learning rates decrease to `5e-6` and `1e-5`, respectively. 
2. The best results are usually not achieved in the last epoch due to training fluctuations.
3. I used the cosine annealing scheduler to reduce learning rates.
4. The drop path rate introduces uncertainty into the training. The same results cannot be got unless the randomness of the training is fixed, by setting fixed seeds.
5. I have just done several rounds of training after reading your comment, and found that some training achieved better results in 2 or 3 metrics than the results reported in the paper. But also one round of training couldn't converge. 
6. You can use Tensorboard to check the training process. `tensorboard --logdir ./tmp/your_saved_model_folder`.

**Comment by FangGet** [comment]
Thanks.
1. useful, I will try that.
2. I also get best results not from last epoch.
3. Cosine annealing has also been tried, but with a much smaller eta lr, I will try it with tips. 1.
4.5.6. understand, thanks again.

**Comment by FangGet** [comment]
done, similar performance, and I notice that you uploaded the training code, close issue.
